### DISCUSSION TITLE
I tried to display images in publication pages in different way, but it still didn't work. I tried a lot of different paths, but it didn't work. What am I missing?

### DISCUSSION THREAD

**Discussion by MarkMaaaaa** [academicpages/academicpages.github.io#2378] [Q&A]
---
layout: archive
title: "Publication"
permalink: /publications/
author_profile: true
excerpt: "Short description of portfolio item number 2 <br/><img src='/images/500x300.png'>"
---
excerpt: "Short description of portfolio item number 2 <br/><img src='/images/500x300.png'>"
{% include base_path %}

<h2>Recent Project</h2>

<div class="publication">
  <p><strong>"Automated Vehicle Planning & Control Test"</strong></p>
  <p><em>Conducted from 09/2020 to Now</em></p>

  
  <img src="project_AV.jpg">
  excerpt: "Short description of portfolio item number 2 <br/><img src='/images/500x300.png'>"
  <p><a href="https://www.sciencedirect.com/science/article/abs/pii/S0968090X22003400">Relative Publication</a></p>
</div>
 excerpt: "Short description of portfolio item number 2 <br/><img src='/images/500x300.png'>"
![c65c570f490b660169d82aca43f86e6](https://github.com/user-attachments/assets/c20819b7-96f5-47c0-8d96-d4285b586398)

{% for post in publications %}
  {% include archive-si
![c65c570f490b660169d82aca43f86e6](https://github.com/user-attachments/assets/b43ce5fc-3f55-4deb-8ab5-41709befbfe5)
ngle.html %}
{% endfor %}

**Comment by rjzupkoii** [comment]
This is most likely a pathing issue, what's the source code for your images look like?

**Reply by wanbint** [reply]
Hi Robert,

I'm having the same issue. I'm working on the "about.md" page and have tried several methods to load the image (listed below), but none of them worked.

![image](https://github.com/user-attachments/assets/bc9543b3-0cf8-49c2-945a-4371eae8c869)



It doesn't seem to be a path issue since the image loads correctly in preview mode.

Best,

**Reply by rjzupkoii** [reply]
@wanbint If the images are loading correctly in preview, but not on the live website that would imply a pathing issue, and that tends to happen since the paths can change between your repository and once things get deployed to github.io

Usually `/images/image.png` will work but if you alter the location that your site deploys to you need to take that into account. Looking at your website I found that the image would be returned when I used the following path: `https://wanbint.github.io/0/images/wbIdealHousing.jpg` so you may want to try `/0/images/wbIdealHousing.jpg`.

**Reply by wanbint** [reply]
Thanks. I tested your code, and it worked on my page.

**Reply by SwagaRuney** [reply]
I seem to be having this issue as well, i'm attempting to understand but i fail to get it to work...

**Comment by scaliwalker** [comment]
I am having the same problem. 
I am working on the research.md of my github.io academic pages template. 

https://github.com/scaliwalker/edoardo_scali.github.io/blob/master/research.md

I can display this image in the preview: 
<img src="../master/images/inoculation_1.jpg" alt="Inoculation of cypress seedling with _Seiridium cardinale_">
But not in on the website. 

I don't understand what is the problem. I successfully loaded images in other pages of the website. But really can't fix this one. 

Can yolu help me with this issue?

**Reply by rjzupkoii** [reply]
It looks like you are using a relative path based upon the path in GitHub:

```html
<img src="../master/images/inoculation_1.jpg" alt="Inoculation of cypress seedling with _Seiridium cardinale_">
```

That should be an absolute path in relation to the file structure of your repository:

```html
<img src="/images/inoculation_1.jpg" alt="Inoculation of cypress seedling with _Seiridium cardinale_">
```

**Reply by scaliwalker** [reply]
Thank you for the help. 
I tried with your suggestion but still having the same problem. 

Are you able to see the image opening this one? https://scaliwalker.github.io/edoardo_scali.github.io/research/

I can only display the jpg correctly in the preview. 

Thanks again for the support!

**Reply by rjzupkoii** [reply]
@scaliwalker It looks like it's on the right track though, I used the inspect function in Chrome to check how the browser is trying to resolve the file and it's looking for `https://scaliwalker.github.io/images/inoculation_1.jpg` when it should be looking for `https://scaliwalker.github.io/edoardo_scali.github.io/images/inoculation_1.jpg`.

Depending on where the code is, you may need to update your `_config.yml`, but for HTML `img` tags you are going to need to use `edoardo_scali.github.io/images/inoculation_1.jpg` for the path.

**Reply by scaliwalker** [reply]
@rjzupkoii thank you so much for pointing it out. 
I have updated the _config.yml basic site setting section like this: 

# Basic Site Settings
locale                   : "en-US"
title                    : "Edoardo Scali"
title_separator          : "-"
name                     : &name "Edoardo Scali"
description              : &description "personal description"
url                      : "https://scaliwalker.github.io" # the base hostname & protocol for your site e.g. "https://mmistakes.github.io"
baseurl                  : "/edoardo_scali.github.io" # the subpath of your site, e.g. "/blog"
repository               : "scaliwalker/edoardo_scali.github.io"

I am still not able to see the image on the website https://scaliwalker.github.io/edoardo_scali.github.io/research/

Does my modification on the setting at least fixed the issue with what the browser is searching for? 

Thank you again for your help!

**Reply by rjzupkoii** [reply]
@scaliwalker The `url` and `baseurl` are used as part of compilation, but since you are using the HTML `img` tag for that image you need to be more explicit about the path. Try the following:

```html
<img src="/edoardo_scali.github.io/images/inoculation_1.jpg" alt="Inoculation of cypress seedling with _Seiridium cardinale_">
```

**Reply by scaliwalker** [reply]
@rjzupkoii 
it worked!!
Thank you so much for your help! :)
